### PR TITLE
Fix/stop connector jobs

### PR DESCRIPTION
**Pull request by SurabhiAngadi**

### CODE REVIEW

File: command-service/src/command/connector_command.py [hunk 1]
```diff
@@ -64,29 +64,30 @@ def _stop_connector_jobs(self, is_masterdata, namespace):
...
-                    else:
-                        print(
-                            f"Error uninstalling job {release_name}: {helm_uninstall_result.stderr.decode()}"
-                        )
-
+            for connector in active_connectors:
+                if connector.dataset_id == dataset_id:
+                    release_name = connector.id
+                    
+                    for job in jobs.splitlines()[1:]:
```
Comment (SanthoshVasabhaktula) [inline command-service/src/command/connector_command.py:71]: 1. No need to compare against dataset as the query filters only dataset specific connectors
2. Filter out only spark jobs from the active connectors
3. No need of nested loops. once alls the jobs are fetch, check if the connector is in the jobs list and then uninstall it

**Review by SanthoshVasabhaktula — CHANGES_REQUESTED** [review]

**Review by anandp504 — APPROVED** [review]
We need to revisit the logic of uninstalling and reinstalling the cronjob for a connector instance. We can just perform a helm upgrade --install as the release name is going to be the instance id, i.e. `datasetId-connectorType`. 
Approved 👍

**Review by anandp504 — APPROVED** [review]
Approved 👍

**Review by anandp504 — DISMISSED** [review]
Approved 👍

**Review by anandp504 — APPROVED** [review]
Approved 👍

**Review by SanthoshVasabhaktula — APPROVED** [review]

**Review by SanthoshVasabhaktula — APPROVED** [review]

**Review by SanthoshVasabhaktula — DISMISSED** [review]

**Review by anandp504 — DISMISSED** [review]
Approved 👍

**Review by HarishGangula — DISMISSED** [review]

**Pull request closed without merging**